readme.md

```diff
@@ -92,6 +92,20 @@ There is a very tiny chance of a race condition if another process starts using
 
 Race conditions in the same process are mitigated against by using a lightweight locking mechanism where a port will be held for a minimum of 15 seconds and a maximum of 30 seconds before being released again.
 
+#### IPv6 vs IPv4 - Port incorrectly returns as free
+```js
+(async () => {
+    // When calling getPort without a host, the underlying system determines if it's IPv6 or IPv4
+    // Even if a port is occupied, under some situations it might incorrectly return as free
```

Comment:
It should explain "what situations".

Reply:
Do we know what situations though?

isn't that dependent of the underlying OS? 
I'm a bit out of my element here when it comes to networking/.